### PR TITLE
Allow setting the published_at field in the API

### PR DESCRIPTION
**Pull request by alexandrebodin**
#### Description of what you did:

Allow setting the published_at field in the API

### CODE REVIEW

**Review by Convly — APPROVED** [review]
LGTM

**Review by petersg83 — APPROVED** [review]
Works well on my side

File: packages/strapi-plugin-content-manager/test/search.test.e2e.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const bedFixtures = [
     sku: 'sleepybed_0152',
     savEmail: 'sav@bed.fr',
     type: 'foam',
-    serialNumber: 2908199401091998,
+    serialNumber: 9999999999999999,
```
Comment (petersg83) [inline packages/strapi-plugin-content-manager/test/search.test.e2e.js:60]: Why was it needed? :)

**Review by petersg83 — APPROVED** [review]
LGTM

**Pull request merged**